packages/amplify_datastore/example/android/app/build.gradle

```diff
@@ -30,6 +30,8 @@ android {
     ndkVersion flutter.ndkVersion
 
     compileOptions {
+        coreLibraryDesugaringEnabled true
```

Comment:
What's this for?

Reply:
This is required by `amplify-android` using DataStore plugin.
https://docs.amplify.aws/lib/datastore/getting-started/q/platform/flutter/#update-the-android-project